com.unity.render-pipelines.universal/Editor/UniversalRenderPipelineAssetEditor.cs

```diff
@@ -342,6 +353,16 @@ void DrawLightingSettings()
                 DrawShadowResolutionTierSettings();
                 EditorGUI.EndDisabledGroup();
 
+                EditorGUI.indentLevel--;
+                EditorGUILayout.Space();
+
+                // Reflection Probes
+                EditorGUILayout.LabelField(Styles.reflectionProbesSettingsText);
+                EditorGUI.indentLevel++;
+                EditorGUILayout.PropertyField(m_ReflectionProbeBlendingProp, Styles.reflectionProbeBlendingText);
+                EditorGUILayout.PropertyField(m_ReflectionProbeBoxProjectionProp, Styles.reflectionProbeBoxProjectionText);
+                EditorGUI.indentLevel++;
```

Comment:
This ident increment seems redundant. Just remove the decrement below?

Reply:
fixed in 3952967